crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -1317,6 +1317,10 @@ end
 #!
 #! Panics if:
 #! - the procedure root is not part of the account code.
+#!
+#! NOTE: This returns true only if the procedure invoked account-restricted kernel APIs (e.g.,
+#! `exec.faucet::mint`) which trigger `authenticate_and_track_procedure`. Procedures that execute
+#! only local MASM instructions will return false even if they were executed.
 export.was_procedure_called
```

Comment:
I think if we stick to the format of `crates/miden-lib/masm_doc_comment_fmt.md`, then this would go into the "General description" part, e.g.:

```
<General description>

Inputs:  [...]
Outputs: [...]

Where:
- ...

Panics if:
- ...

Locals:
- ...

Invocation: ...
```

Everything below the description is basically well-categorized into `Where`, `Panics if`, etc. (If agreed, we could update the format guide at some point to include `Locals`, but requires a bit of discussion).